index.html

```diff
@@ -143,8 +143,11 @@ <h1>
               <div class="image-container">
                 <img
                   class="img-fluid"
-                  src="img/ic_launcher-web.png"
+                  src="img/shot_1.webp"
```

Comment:
It might look nicer if the screenshot were showing an in-app view IMO. Other than that, it is approvable.

Reply:
I agree, a screenshot of maybe the home page when the app is opened would be preferred over the web UI of trebleshot. [This](https://trebleshot.monora.org/img/shot3.webp) screenshot seems to be a better candidate.

Reply:
Okay, I am trying to do changes according to you.